### DISCUSSION TITLE
Command Palette

### DISCUSSION THREAD

**Discussion by karniv00l** [leoafarias/sidekick#250] [Ideas]
Hi 👋 

I just thought that could be nice to use something like https://github.com/TNorbury/command_palette instead of a plain search field, that way some of us will feel like home ;)

**Comment by aguilaair** [comment]
I like it!

**Comment by leoafarias** [comment]
@karniv00l I like this command palette, however, the current field is a search, not a command shortcut. We do support some command shortcuts and maybe can add more.
